jest.config.js

```diff
@@ -1,5 +1,5 @@
-{
+module.exports = {
```

Comment:
Possible undefined variable: 'module' is not defined.

If this is intended to be a global variable, you can supress this warning by adding a  `/*global [VARIABLE NAME] */` comment on this file or set it under the `globals` section of your `.eslintrc` configuration.

[Read more about it here.](http://eslint.org/docs/rules/no-undef)